recipe/meta.yaml

```diff
@@ -36,36 +34,34 @@ requirements:
     - geotiff
     - giflib   # [not win]
     - hdf4
-    - hdf5
-    - icu
+    - hdf5 1.12.1
     - jpeg
     - json-c   # [not win]
     - kealib
     - lerc
-    - libcurl
+    - libcurl 7.88.1
     - libdeflate 1.17
     - libiconv
     - libkml
-    - libnetcdf
+    - libnetcdf 4.8.1
     - libpng
-    - libpq
+    - libpq 12.15
     - libspatialite
     - libtiff
     - libuuid  # [linux]
     - libwebp-base
     - libxml2 2.10
     - lz4-c
     - openjpeg
-    - openssl
+    - openssl 3.0.8
```

Comment:
Why not 3.0.9? I see it on defaults for every platform. Applies to all other occurrences.

Reply:
I can use 3.0.9 but it won't make any difference. I used 3.0.8 because that's what we used so far and completely forgot that we now have 3.0.9.